modules/cnf-topology-aware-lifecycle-manager-cluster-logging-6x-update-policygentemplate.adoc

```diff
@@ -0,0 +1,139 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-talm-updating-managed-policies.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="talm-cluster-logging-6x-update-policygentemplate_{context}"]
+= Removing Cluster Logging Operator artifacts from deployed clusters with PolicyGenTemplate CRs
```

Comment:
```suggestion
= Removing Cluster Logging Operator artifacts with PolicyGenTemplate 
```

Same comment/Question: Can this heading be shorten like this by adding the details in the intro para below?